plugins/system_upgrade.py

```diff
@@ -495,8 +495,14 @@ def check_upgrade(self):
         # Delete symlink ASAP to avoid reboot loops
         dnf.yum.misc.unlink_f(MAGIC_SYMLINK)
         if not self.state.upgrade_status == 'ready':
-            raise CliError(  # Translators: do not change "reboot" here
-                _("use 'dnf system-upgrade reboot' to begin the upgrade"))
+            command = self.state.upgrade_command
+            if command == 'offline-upgrade':
+                msg = _("use 'dnf offline-upgrade reboot' to begin the upgrade")
+            elif command == 'offline-distrosync':
+                msg = _("use 'dnf offline-distrosync reboot' to begin the upgrade")
+            else:
+                msg = _("use 'dnf system-upgrade reboot' to begin the upgrade")
```

Comment:
Can you please use {command} parameter in this message, too?

Reply:
Resolved